network-api/networkapi/templates/fragments/buyersguide/content_category_links.html

```diff
@@ -1,10 +1,12 @@
-{% with content_categories=page.get_content_categories editorial_content_index=page.get_editorial_content_index  %}
-  {% for content_category in content_categories %}
-    {% with classes="tw-h6-heading tw-text-blue-80 tw-mr-2 tw-py-1 tw-mb-0" %}
-      <a class="{{ classes }} {{ extra_classes }}" href="/privacynotincluded/articles/">{{ content_category.title|title }}</a>
-      {% if not forloop.last %}
-        <span class="{{ classes }} {{ extra_classes }}">/</span>
-      {% endif %}
-    {% endwith %}
-  {% endfor %}
-{% endwith %}
+{% if content_categories %}
+  <div class="tw-h6-heading tw-mr-2 tw-mb-0 {{ extra_classes }}">
+    {% for content_category in content_categories %}
+        <span>
+          {{ content_category.title }}
```

Comment:
```suggestion
          {{ content_category.localized.title }}
```

Hey Tibor! 
I am thinking that we may need to update the template to render something such as `category.localized.title`.

The reason I bring this up, is because even after translating a category and then visiting a localized article page, it appears that the English category is still being rendered.

For example, I translated the category "Agency"'s French title to "Test Agency". 

![image](https://user-images.githubusercontent.com/18314510/208558024-0f06c2cb-700e-4d4c-a70b-047ffadb1d09.png)

However, when visiting a French article page, it is still rendering the English title. Even after running sync translations, etc:

<img width="720" alt="Screenshot 2022-12-19 at 5 16 05 PM" src="https://user-images.githubusercontent.com/18314510/208558672-e7276297-1e30-45ca-a414-e1b1c094791d.png">

Reply:
Thanks for catching this. I must admit, I did not really test the frontend at all 🤦‍♂️ 

In general though, I would highly recommend against the use of `.localize` in templates. This is because that accessing that property will likely trigger an additional database query. As of now, this property is also not cached, so every access triggers another database query: https://github.com/wagtail/wagtail/blob/cbe9dc729b7cc92229b535c9bedd719b1a12c9e8/wagtail/core/models/__init__.py#L782-L796
This, I believe, is probably one of the reasons we end up with 80 queries on basically empty pages. 

As an alternative strategy, I would recommend using the `get_context` or other retrieval method on the page that ensures that the data used in the template is using the correct locale. In the case of this model, the `get_content_categories` seems like a good location to me.

Reply:
This is addressed in 22a8e613535711c288c570066e05a99f9b26e354